Abstract update (#8)
arfc/2020-fairhurst-hydrogen-production: ans-abstract/abstract.tex

	\label{fig:ghg}
\end{figure}

One possible solution to reduce carbon emissions, and even achieve a net zero carbon production, is to develop a hydrogen economy as the state of California is currently doing \cite{brown_economic_2013}. 

> Although using hydrogen does not produce CO$_2$, all hydrogen production methods are only as carbon-free as the source of energy they rely on (electric, heat, or both) to drive the process.

Reviewer: ```suggestion
Although using hydrogen does not produce CO$_2$, any hydrogen production method is only as carbon-free as the source of energy it relies on (electric, heat, or both).
```
